docs/export-format.md

```diff
@@ -0,0 +1,158 @@
+# Briefcase export CSV format
+
+## Format parameters
+
+| Parameter               | Value | Description |
+| ----------------------- | ----- | ----------- |
+| Separator               | `,`   | Comma character |
+| Quoted                  | `"`   | Double quote character. See the [Values block columns](#values-block-columns) section to know which fields get quoted |
+| Header                  | Yes   | Every file will have a header with the respective column names |
+| Empty columns           | Yes   | See the "Values block columns" section to know which fields get empty columns |
+| Encoding of null values | No    | Nulls are just empty columns and, sometimes, empty strings |
+
+## Main submission output file
+
+The main submission out file has values from any non repeating group field found at the topmost level of a form's model and other metadata.
+
+### Structure of the file:
+
+| Column block | Description |
+| ------------ | ----------- |
+| Fixed block  | Holds a column for the submission date |
+| Values block | Holds columns with the values of topmost model fields and non repeating groups |
+| Fixed block  | Holds a column for the submission key and, when the form is encrypted, a content validation flag |
+
+### Fixed block columns
+
+| Column name    | Optional | Quoted | Description |
+| -------------- | -------- | ------ | ----------- |
+| SubmissionDate | No       | Yes    | Holds the submission date. See the [Values block columns](#values-block-columns) section to learn how it is encoded |
+| KEY            | No       | No     | Holds the `instanceID` of each submission |
+| isValidated    | Yes      | No     | It's only present if the form is encrypted. It holds `true` if the cryptographic signature has been validated, `false` otherwise |
+
+## Repeat output file
+
+Each repeat group will produce an output CSV file.
+
+### Structure of the file:
+
+| Column block | Description    |
+| ------------ | -------------- |
+| Values block | Holds columns with the values repeat group fields |
+| Fixed block  | Holds three identifying columns |
+
+### Fixed columns:
+
+| Column name         | Optional | Description |
+| ------------------- | -------- | ----------- |
+| PARENT_KEY          | No       | Identifies the parent row |
+| KEY                 | No       | Identifies this row |
+| SET-OF-{GROUP NAME} | No       | Identifies the group this row belongs to |
+
+### The parent key `PARENT_KEY`
+
+|                        | Pattern                                                                            | Example                                               |
+| ---------------------- | ---------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| Topmost level pattern  | `{INSTANCE ID}`                                                                    | `uuid:00000000-0000-0000-0000-000000000000`           |
```

Comment:
Maybe parent pattern and children pattern?

Reply:
Same argument as before: a child can be a parent of some other field. I'm not sure about changing the semantics.

As an alternative, we could use "root" and "node", but I'd suggest changing the code to use the same language. These dissonances between the domain and the source code are confusing down the road.